.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: java
 jdk:
   - openjdk6
+    script: mvn test -DlegacyJdk=true
```

Comment:
I think you will also want to do:

```yaml
install: true
```
to bypass mvn install (which mvn test will do anyways).